src/charm.py

```diff
@@ -1459,15 +1445,33 @@ def _was_restore_successful(self, container: Container, service: ServiceInfo) ->
             logger.debug("Restore check early exit: Patroni has not started yet")
             return False
 
+        restoring_backup = self.app_peer_data.get("restoring-backup")
+        restore_timeline = self.app_peer_data.get("restore-timeline")
+        restore_to_time = self.app_peer_data.get("restore-to-time")
+        # Sometimes this check fails to connect to the PostgreSQL. Not happened on VM, so it's a k8s-specific try-catch
+        # try:
+        #     current_timeline = self.postgresql.get_current_timeline()
+        # except PostgreSQLGetCurrentTimelineError:
+        #     logger.debug("Restore check early exit: can't get current wal timeline")
+        current_timeline = self.postgresql.get_current_timeline()
+
         # Remove the restoring backup flag and the restore stanza name.
         self.app_peer_data.update({
             "restoring-backup": "",
             "restore-stanza": "",
             "restore-to-time": "",
+            "restore-timeline": "",
         })
         self.update_config()
         self.restore_patroni_on_failure_condition()
-        logger.info("Restore succeeded")
+
+        logger.info(
+            "Restored"
+            f"{' to ' + restore_to_time if restore_to_time else ''}"
+            f"{' from timeline ' + restore_timeline if restore_timeline and not restoring_backup else ''}"
+            f"{' from backup ' + self.backup._parse_backup_id(restoring_backup)[0] if restoring_backup else ''}"
```

Comment:
Also, here: will the charm reach this code without a backup ID?

Reply:
If user starts restore with `backup-id` pointing to the `action: restore`, so there wouldn't be a `restoring_backup` variable and only the `restore_timeline` with `restore_to_time`. But for the restore, backup-id is now required for user to specify.